go/spikes_input/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
```

Comment:
Is there opencv for go?

Reply:
It appears that there is something, not sure if its what we need, maybe try again with normal go

Reply:
Make it work in go